eclipse-sirius/sirius-components
# [937] [938] [1019] Add diagram to SVG service

### Type of this PR 


### Issue(s)

Feature: #937 & #938 & #1019

### What does this PR do?

This PR adds a new service,  SVGExportService,  supporting the SVG export of a diagram.

### How to test this PR?

Add the following code snippet to DiagramEventFlux#diagramRefreshed and refresh a diagram. The svg string should be displayed on the backend console.

```
        SVGExportService export = new SVGExportService("http://localhost:8080"); //$NON-NLS-1$
        String svg = export.toSVG(this.currentDiagram);
        System.out.println(svg);
```

### Checklist

- [ ] New React components are availables in storybook
